### PR TITLE
VAL-555 Adding bulk preview.

### PR DESCRIPTION
**Pull request by aerwin3**
https://mailgun.atlassian.net/browse/VAL-555

### CODE REVIEW

File: source/api-email-validation.rst [hunk 1]
```diff
@@ -330,3 +330,195 @@ Sample Response:
...
+
+
+Bulk Validation Preview
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+    Bulk validation preview performs a free analysis of a list of email addresses allowing
+    you to make an informed decision to run a complete bulk validation or not. Given a preview name
+    and an uploaded file of email addresses, a preliminary validation run will be preformed. The results of the preview
+    will be, on average, and estimate of the deliverability and risk of the emails provide. This evaluation is based
```
Comment (jamiramontez) [inline source/api-email-validation.rst:342]: replace "and estimate" with "an estimate"

File: source/api-email-validation.rst [hunk 2]
```diff
@@ -330,3 +330,195 @@ Sample Response:
...
+    on a statistical sampling of the list provided
```
Comment (jamiramontez) [inline source/api-email-validation.rst:343]: Punctuation please :)

File: source/api-email-validation.rst [hunk 3]
```diff
@@ -330,3 +330,195 @@ Sample Response:
...
+
+
+.. note::
+    Its important to upload as `multi-part/form-data` where the file is defined by `file`.
```
Comment (jamiramontez) [inline source/api-email-validation.rst:347]: Grammar..."It's" not "Its"

File: source/api-email-validation.rst [hunk 4]
```diff
@@ -330,3 +330,195 @@ Sample Response:
...
+=====================    ===========    ============================================================================================================
+Field                    Type           Description
+=====================    ===========    ============================================================================================================
+id                       string         list_id name given when the list was initially created
+created_at               string         Date/Time that the request was initiated
+quantity                 integer        number of total items in the list to be previewed
+status                   string         current state of the list validation request. (``preview_processing``, ``preview_complete``)
+valid                    bool           a boolean to represent if the list is valid
+summary                  collection     summary of the validations in the list provided
+result                   array          nested results averaged. (``deliverable``, ``undeliverable``, and `unknown`)
```
Comment (jamiramontez) [inline source/api-email-validation.rst:431]: need double ticks for "unknown"

File: source/api-email-validation.rst (hunk 1 above)
Comment (aerwin3) [reply source/api-email-validation.rst:342]: updated.

File: source/api-email-validation.rst (hunk 4 above)
Comment (aerwin3) [reply source/api-email-validation.rst:431]: updated.

**Pull request merged**